saas/models/saas_template.py

```diff
@@ -214,3 +214,18 @@ def create_db(self, db_name, key_values):
         self.operator_id.with_delay().build_post_init(build, self.template_id.build_post_init, key_value_dict)
 
         return build
+
+
+class IrActionsServer(models.Model):
+    _name = 'ir.actions.server.custom.default'
+    _inherit = 'ir.actions.server'
+
+    DEFAULT_PYTHON_CODE = """# Available variables:
```

Comment:
:-1: You don't need to create a table just to introduce a constant

Reply:
ok